go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/e-dard/netbug v0.0.0-20151029172837-e64d308a0b20
 	github.com/elazarl/go-bindata-assetfs v1.0.1
 	github.com/facebookincubator/nvdtools v0.1.6-0.20231010102659-d14ce526f176
-	github.com/fatih/color v1.13.0
+	github.com/fatih/color v1.15.0
```

Comment:
Why do we see other deps changed?

Reply:
They are probably generated by `go mod tidy` which should be executed every time dependencies are updated.

Reply:
OK. I also see @mostlikelee approved, so I'm fine with it.